### PR TITLE
[ACCESS] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by AndreiMiloiu** [comment]
Any news?

**Comment by AndreiMiloiu** [comment]
Is anything to change? Or is it good?

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/cpl/access/lang/ro-RO.rc [hunk 1]
```diff
@@ -39,262 +39,262 @@ CAPTION "Sunet"
...
 IDD_PROPPAGEGENERAL DIALOGEX 0, 0, 246, 220
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Generale"
+CAPTION "General"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    GROUPBOX "Resetare automată", -1, 6, 11, 234, 47
-    AUTOCHECKBOX "Oprește facilitățile din Accesi&bilitate după o perioadă inactivă de:",
+    GROUPBOX "Revenire automată", -1, 6, 11, 234, 47
+    AUTOCHECKBOX "Se a&plică modificările doar pentru această sesiune Windows",
```
Comment (GeoB99) [inline dll/cpl/access/lang/ro-RO.rc:98]:
```suggestion
    AUTOCHECKBOX "Se a&plică modificările doar pentru această sesiune ReactOS",
```

**Pull request merged**